src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -1,450 +0,0 @@
-    tcf_system_relationships: List[TCFVendorRelationships] = []
-
-
-def get_legal_basis_override_subquery(db: Session) -> Alias:
-    """Subquery that allows us to globally override a purpose's legal basis for processing.
-
-    Original legal basis for processing is returned where:
-    - feature is disabled
-    - declaration's legal basis is not flexible
-    - no legal basis override specified
-
-    Null is returned where:
-    - Purpose is excluded (this will effectively remove the purpose from the Experience)
-
-    Otherwise, we return the override!
-    """
-    config_proxy = ConfigProxy(db)
-    if not config_proxy.consent.override_vendor_purposes:
-        return db.query(
-            PrivacyDeclaration.id,
-            PrivacyDeclaration.legal_basis_for_processing.label(
-                "overridden_legal_basis_for_processing"
-            ),
-        ).subquery()
-
-    return (
-        db.query(
-            PrivacyDeclaration.id,
-            case(
-                [
-                    (
-                        TCFPurposeOverride.is_included.is_(False),
-                        None,
-                    ),
-                    (
-                        PrivacyDeclaration.flexible_legal_basis_for_processing.is_(
-                            False
-                        ),
-                        PrivacyDeclaration.legal_basis_for_processing,
-                    ),
-                    (
-                        TCFPurposeOverride.required_legal_basis.is_(None),
-                        PrivacyDeclaration.legal_basis_for_processing,
-                    ),
-                ],
-                else_=TCFPurposeOverride.required_legal_basis,
-            ).label("overridden_legal_basis_for_processing"),
-        )
-        .outerjoin(
-            TCFPurposeOverride,
-            TCFPurposeOverride.purpose == PrivacyDeclaration.purpose,
-        )
-        .subquery()
-    )
-
-
-def get_tcf_base_query_and_filters(
-    db: Session,
-) -> Tuple[Query, BinaryExpression, BinaryExpression]:
-    """
-    Returns the base query that contains the foundations of the TCF Experience as well as
-    two filters to further refine the query when building the Experience.
-
-    Rows show up corresponding to systems with GVL data uses and Legal bases of Consent or Legitimate interests.
-    AC systems are also included here.
-    Purpose overrides are applied at this stage which may suppress purposes or toggle the legal basis.
-    """
-    legal_basis_override_subquery = get_legal_basis_override_subquery(db)
-
-    consent_legal_basis_filter: BinaryExpression = (
-        legal_basis_override_subquery.c.overridden_legal_basis_for_processing
-        == LegalBasisForProcessingEnum.CONSENT
-    )
-    legitimate_interest_legal_basis_filter: BinaryExpression = (
-        legal_basis_override_subquery.c.overridden_legal_basis_for_processing
-        == LegalBasisForProcessingEnum.LEGITIMATE_INTEREST
```

Comment:
TCF base query now moved to Plus!  We need to double check that Purpose overrides functionality still works as expected.